src/utils/url.ts

```diff
@@ -1,6 +1,11 @@
 import Bugsnag from "@bugsnag/js";
+
 import { VALID_URL_PROTOCOLS } from "settings";
 
+export interface JoinVenueOptions {
+  hasVenueEntrance?: boolean;
+}
```

Comment:
[NB] I'm not sure if I have strong feelings one way or the other about this, but usually I tend to keep the types close to the function/etc that they relate to. Though I believe other patterns/conventions keep them at the top of the file, as you've done here. Don't need to make changes here.. just wanted to point it out/bring it up as something we should probably consider/standardise on as a team.

Reply:
I agree, we should discuss this and add it in the documentation PR if it's missing there.

Reply:
Sounds good, can you please raise this in `#tech`